@xen-orchestra/backups/_VmBackup.js

```diff
@@ -401,6 +410,24 @@ class VmBackup {
     this._fullVdisRequired = fullVdisRequired
   }
 
+  async _healthCheck() {
+    const settings = this._settings
+
+    if (settings.healthCheck === undefined) {
+      return
+    }
+
+    // check if current VM has tags
+    const { tags } = this.vm
+    const intersect = settings.healthCheck.tags.filter(t => tags.includes(t))
```

Comment:
What you need is [`Array#some()`](https://devdocs.io/javascript/global_objects/array/some).